### DISCUSSION TITLE
A simple example of a tcl function within SqliteStudio

### DISCUSSION THREAD

**Discussion by PatriaMinas** [pawelsalawa/sqlitestudio#4324] [General]
This is a simple example of a function that was created in SqliteStudio with the sql language (I'm sorry the title was wrong as tcl, but I want to learn tcl).
It's very difficult to find any example, so I'm proposing to add here.
The first image is the function being created. The second image is the test of the function with the sql select command.
If this post is not correct, please remove it from this thread.
Thank you.

![im1](https://user-images.githubusercontent.com/95147314/144502596-332cd1d2-e564-4095-a9c4-7c8b8d44f66d.png)

Testing the function:

![im2](https://user-images.githubusercontent.com/95147314/144502627-09b4cdba-1f5b-48a0-9fa0-c480ed2e718a.png)

**Comment by pawelsalawa** [comment]
There is a page with some Tcl and Qt scripts on https://github.com/pawelsalawa/sqlitestudio/wiki/Scripts_repository

**Comment by PatriaMinas** [comment]
I'm sorry the title was wrong as tcl, but I want to learn tcl. I want to make this example script so people can see how simple it is to do in sql. The post has been fixed.
Thank you.
